third_party/ibis/ibis_impala/api.py

```diff
@@ -33,19 +33,37 @@ def impala_connect(
     database="default",
     auth_mechanism="PLAIN",
     kerberos_service_name="impala",
+    use_ssl=False,
+    timeout=45,
+    ca_cert=None,
+    user=None,
+    password=None,
+    pool_size=8,
+    hdfs_client=None
 ):
     auth_mechanism = (auth_mechanism, "PLAIN")[auth_mechanism is None]
     database = (database, "default")[database is None]
     port = (port, 10000)[port is None]
     kerberos_service_name = (kerberos_service_name, "impala")[
         kerberos_service_name is None
     ]
+    use_ssl = (use_ssl, False)[use_ssl is None]
+    timeout = (timeout, 45)[timeout is None]
+    pool_size = (pool_size, 8)[pool_size is None]
```

Comment:
I took as a reference how the code was already handling "host" parameters. Since the default value is None then it is not added in this section; for the other variables the default values are boolean or Int, and that is why it validates if the value is None to assign the default value. If it requires a change just let me know.